csds-client/client/response_with_nodeid_test.json

```diff
@@ -0,0 +1,42 @@
+{
+  "config": [
+    {
+      "node": {
+        "id": "test_nodeid",
+        "metadata": {
+          "TRAFFICDIRECTOR_GCP_PROJECT_NUMBER": "fake_project_number",
+          "TRAFFICDIRECTOR_NETWORK_NAME": "fake_network_name",
+          "TRAFFIC_DIRECTOR_XDS_STREAM_TYPE": "test_stream_type1"
```

Comment:
I thought we decided to change this to XDS_STREAM_TYPE to be generic?

Reply:
Sorry about that. Forgot to modify the test input. Modified now.